tests/test_indexing.py

```diff
@@ -647,3 +650,123 @@ def test_random_with_replace(self):
         pairs2 = index_cl2.index(self.a)
         assert len(pairs2) == 1000
         assert not pairs2.is_unique
+
+
+# NeighbourhoodBlock
+
+class TestNeighbouroodBlock(TestData):
```

Comment:
Move this class to a new Python file (neighbourhoodblock_test.py) in the contrib folder. 

Check out https://github.com/J535D165/recordlinkage/tree/master/recordlinkage/contrib/compare for inspiration.

Reply:
Also note:  the NeighbourhoodBlock instance is still included in get_test_algorithms (in tests/test_indexing.py).  Happy to delete it if you wish.